vtex-apps/io-documentation
# Fixed wording: "Installment" -> "Installation"

Corrected the title wording and related references from "installment" to "installation"

**What problem is this solving?**

The title was wrong. "Installment" was used where "Installation" was meant.

**How should this be manually tested?**

- Check if [old URL](https://vtex.io/docs/recipes/development/vtex-io-cli-installment-and-command-reference/) is removed
- Check if [new URL](https://vtex.io/docs/recipes/development/vtex-io-cli-installation-and-command-reference/) works
- Check if side menu has correct wording

**Screenshots or example usage:**

Wrong URL:
![Screen Shot 2020-04-07 at 22 03 31](https://user-images.githubusercontent.com/2094877/78733391-b2db0f00-791b-11ea-8a1b-e956e3e1ce36.png)

Wrong Title:
![Screen Shot 2020-04-07 at 22 03 43](https://user-images.githubusercontent.com/2094877/78733400-b7072c80-791b-11ea-91e3-016ba43920fe.png)

Wrong Side Menu:
![Screen Shot 2020-04-07 at 22 03 22](https://user-images.githubusercontent.com/2094877/78733405-ba021d00-791b-11ea-843b-2f6d4b5247a5.png)
